src/com/github/miachm/sods/Style.java

```diff
@@ -14,13 +14,25 @@
     private Color fontColor;
     private Color backgroundColor;
     private int fontSize = -1;
+    private boolean border = false;
+    private boolean borderTop = false;
+    private boolean borderBottom = false;
+    private boolean borderLeft = false;
+    private boolean borderRight = false;
+    private boolean wrap = false;
+    
+    /**
+     * Default border properties.
+     */
+    public static final String BORDER_PROPERTIES = "0.035cm solid #000000";
```

Comment:
You are applying a "fixed" value to the border, a fixed value that you can not alter in any way. That's not addecuate.

I think it would be better to create a new class (CellBorder) and put the characteristics there (width length, type of border, color...).